<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogsFilterWidget</class>
 <widget class="QWidget" name="LogsFilterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>27</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="verboseButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../Resources.qrc">
       <normaloff>:/assets/images/levels/verbose.png</normaloff>:/assets/images/levels/verbose.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="debugButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../Resources.qrc">
       <normaloff>:/assets/images/levels/debug.png</normaloff>:/assets/images/levels/debug.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="infoButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../Resources.qrc">
       <normaloff>:/assets/images/levels/info.png</normaloff>:/assets/images/levels/info.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="warningButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../Resources.qrc">
       <normaloff>:/assets/images/levels/warning.png</normaloff>:/assets/images/levels/warning.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="errorButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../Resources.qrc">
       <normaloff>:/assets/images/levels/error.png</normaloff>:/assets/images/levels/error.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>TIcker:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="tickerLineEdit"/>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../Resources.qrc"/>
 </resources>
 <connections/>
</ui>
